bundles/org.openhab.binding.gree/README.md

```diff
@@ -0,0 +1,157 @@
+|               |           | The number of speeds depends on the Air Conditioner model.                                        |
+
+
+When changing the mode the air conditioner will be turned on (unless is off is selected).
+
+## Full Example
+
+### Generic
+
+Things:
+
+```
+Thing gree:airconditioner:a1234561 [ ipAddress="192.168.1.111", refresh=2 ]
+```
+
+Items:
+
+```
+Switch AirconPower                  { channel="gree:airconditioner:a1234561:power" }
+Number AirconMode                   { channel="gree:airconditioner:a1234561:mode" }
+Switch AirconTurbo                  { channel="gree:airconditioner:a1234561:turbo" }
+Switch AirconLight                  { channel="gree:airconditioner:a1234561:light" }
+Number AirconTemp "Temperature [%.1f °C]" {channel="gree:airconditioner:a1234561:temperature" }
+Number AirconTempSet                { channel="gree:airconditioner:a1234561:temperature" }
+Number AirconSwingVertical          { channel="gree:airconditioner:a1234561:swingvertical" }
+Number AirconFanSpeed               { channel="gree:airconditioner:a1234561:windspeed" }
+Switch AirconAir                    { channel="gree:airconditioner:a1234561:air" }
+Switch AirconDry                    { channel="gree:airconditioner:a1234561:dry" }
+Switch AirconHealth                 { channel="gree:airconditioner:a1234561:health" }
+Switch AirconPowerSaving            { channel="gree:airconditioner:a1234561:powersave" }
+```
+
+Sitemap:
+
+This is an example of how to set up your sitemap.
+
+```
+Frame label="Controls"
+{
+   Switch item=AirconPower label="Power" icon=switch
+   Switch item=AirconMode label="Mode" mappings=[0="Auto", 1="Cool", 2="Dry", 3="Fan", 4="Heat"]
+   Setpoint item=AirconTemp label="Set temperature" icon=temperature minValue=16 maxValue=30 step=1
+}
+Frame label="Fan Speed"
+{
+   Switch item=AirconFanSpeed label="Fan Speed" mappings=[0="Auto", 1="Low", 2="Medium Low", 3="Medium", 4="Medium High", 5="High"] icon=fan
+}
+Frame label="Fan-Swing Direction"
+{
+   Switch item=AirconSwingVertical label="Direction" mappings=[0="Off", 1="Full", 2="Up", 3="Mid-up", 4="Mid", 5="Mid-low", 6="Down"] icon=flow
+}
+Frame label="Options"
+{
+   Switch item=AirconTurbo label="Turbo" icon=fan
+   Switch item=AirconLight label="Light" icon=light
+   Switch item=AirconAir label="Air" icon=flow
+   Switch item=AirconDry label="Dry" icon=rain
+   Switch item=AirconHealth label="Health" icon=smiley
+   Switch item=AirconPowerSaving label="Power Saving" icon=poweroutlet
+}
+```
+
+### Google Home Assistant
+
+This example shows who to make the GREE Air Conditioner controllable by Google HA (A/C mode + temperature)
+
+Items:
+
+```
+Group Gree_Modechannel                 "Gree"        { ga="Thermostat" } // új Gree bindinggal
+// Gree Klíma
+    Switch   GreeAirConditioner_Powerchannel           "Aircon"                                        {channel="greeair:greeairthing:a1234561:powerchannel", ga="Switch"}
```

Comment:
Looks like here (and below) should change the channel name to a new format.
```suggestion
    Switch   GreeAirConditioner_Powerchannel           "Aircon"                                        {channel="gree:airconditioner:a1234561:power", ga="Switch"}
```

Reply:
fixed